sdk/appcenter-data/src/main/java/com/microsoft/appcenter/data/Data.java

```diff
@@ -598,13 +598,20 @@ public void onCallFailed(Exception e) {
         mPendingCalls.put(result, cosmosDbCall);
     }
 
-    private synchronized <T> void callCosmosDbListApi(
+    public synchronized <T> void callCosmosDbListApi(
```

Comment:
Why is it public?

Reply:
I need to call this method from other class. Did not find anything similar to C# internal.

Reply:
From which class? Put it in the same package and use no keyword (package private). If not in same package we need to talk why.

Reply:
Discussed offline the pattern of using an interface which is now part of the pr.